src/Process/OriginalPhpProcess.php

```diff
@@ -89,7 +89,7 @@ private static function shallExtendEnvironmentWithXdebugMode(): bool
         }
 
         // The last case: Xdebug 3+ running inactive.
-        return ini_get_unsafe('xdebug.mode') === '';
+        return ini_get_unsafe('xdebug.mode') !== 'coverage';
```

Comment:
This should be fine since [Xdebug 2 isn't supported with PHP 7.4](https://xdebug.org/docs/compat)